sc/src/main/java/uk/ac/ebi/atlas/download/FileDownloadController.java

```diff
@@ -91,4 +91,44 @@ public FileDownloadController(ExperimentFileLocationService experimentFileLocati
 
         zipOutputStream.close();
     }
+
+    @RequestMapping(value = "experiments/download/zip",
+            method = RequestMethod.GET,
+            produces = "application/zip")
+    public void
+    downloadMultipleExperimentsArchive(HttpServletResponse response,
+                    @RequestParam(value= "accession", defaultValue = "") List<String> accessions,
+                    @RequestParam(value = "accessKey", defaultValue = "") String accessKey) throws IOException {
```

Comment:
`accessKey`should be removed in this endpoint, too. Also, if requesting multiple experiments, for which one would the key be for?